specification/containerservice/Fleet.Management/tspconfig.yaml

```diff
@@ -34,3 +34,11 @@ options:
     generate-fakes: true
     head-as-boolean: true
     inject-spans: true
+  "@azure-tools/typespec-ts":
```

Comment:
JS generation failure is expected and this is an known issue: https://github.com/Azure/azure-sdk-tools/pull/10034.

Reply:
how does the issue linked relates to adding this?
are we not generating the javascript sdk, but generating the typescript one instead?

Reply:
Sorry my fault what i mean is typescrip generation which usually is JavaScript generation.

Reply:
Should be fixed now.